Source/WebCore/Modules/indexeddb/IDBTransaction.h

```diff
@@ -247,6 +248,7 @@ class IDBTransaction final : public ThreadSafeRefCounted<IDBTransaction>, public
     RefPtr<DOMException> m_domError;
 
     RefPtr<IDBOpenDBRequest> m_openDBRequest;
+    WeakHashSet<IDBRequest, WeakPtrImplWithEventTargetData> m_cursorRequests;
```

Comment:
We don't seem to ever remove from this map. Is relying on object destruction really the right thing to do here?

Reply:
Confirmed with @rniwa that `remove()` is not necessary for `WeakHashSet` as it does cleanup after 2n operations.